sktime/dists_kernels/scipy_dist.py

```diff
@@ -0,0 +1,84 @@
+# -*- coding: utf-8 -*-
+"""
+Interface module to scipy.spatial's pairwise distance function cdist
+    exposes parameters as scikit-learn hyper-parameters
+"""
+
+__author__ = ["fkiraly"]
+
+import pandas as pd
+
+from scipy.spatial.distance import cdist
+
+from sktime.dists_kernels._base import BasePairwiseTransformer
+
+
+class ScipyDist(BasePairwiseTransformer):
```

Comment:
minor:
I'd like to have other people's perspective on the name ScipyDist. To me, this sounds like an elementwise transformer to me, rather than a pairwise transformer